staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_docker.go

```diff
@@ -204,6 +210,11 @@ func (o *CreateSecretDockerRegistryOptions) Complete(f cmdutil.Factory, cmd *cob
 		return err
 	}
 
+	for i := range o.FileSources {
```

Comment:
Can you write some unit tests for using `--from-file` in different ways?

I don't know if you would be able to test the whole command in a unit test, but at least the Complete() func could be tested to make sure it correctly adds the default key when one is missing.

Reply:
@brianpursley added a new unit test for the whole command when a file is specified. please review again.